misbehave.html

```diff
@@ -55,6 +55,17 @@
         behaviorDefinition[key] = thisPrototype[key];
       }
 
+      behaviorDefinition.injectFrom = function(base, target) {
+        var baseElement = document.createElement(base);
+        // Grab an instance of the base template!
+        var baseTemplateInstance = Polymer.Base.instanceTemplate(Polymer.dom(baseElement).node._template);
+        
+        // Remove what might have been in the target
+        Polymer.dom(target).innerHTML = "";
```

Comment:
I made that decision for assurance of expecting the target element to look exactly like the base's template, good thoughts, thinking about the <content> approach.